azext_iot/central/models/device.py

```diff
@@ -31,3 +31,14 @@ def _parse_device_status(self) -> DeviceStatus:
             return DeviceStatus.registered
 
         return DeviceStatus.provisioned
+
+    def get_device_registration_info(self):
```

Comment:
nit - rename to `get_registration_info`, since the object itself is a device its evident that we're getting device registration info.